docs/README.md

```diff
@@ -0,0 +1,59 @@
+# How to document a Giant Swarm product
+
+We provide product-specific documentation inside the repository of the product, using a top-level directory named `docs`.
+
+The content within this directory is intended primarily for reading on the `giantswarm.io` website. To enable useful rendering in that context, please adhere to the following guidelines.
```

Comment:
Yes, I generally agree, just wanted it to be clear that there will be people looking at it through github and it should at least look decent in there.